docs/source/changelog.md

```diff
@@ -12,41 +12,21 @@ changes in pull requests], this list should be updated.
 [development releases]: https://hub.jupyter.org/helm-chart/#development-releases-jupyterhub
 [breaking changes in pull requests]: https://github.com/jupyterhub/zero-to-jupyterhub-k8s/pulls?q=is%3Apr+is%3Aclosed+label%3Abreaking
 
-### Default image registry moved to [quay.io](https://quay.io)
+### Default image registry changed to Quay.io
 
-We have moved the registry where we publish our docker images from [Docker Hub](https://hub.docker.com)
-to [Quay.io](https://quay.io). This move is to ensure our users are not [throttled by Docker Hub](https://docs.docker.com/docker-hub/download-rate-limit/),
-and us maintainers don't have to apply for 'sponsored OSS Project' from docker each year. This
-should have no material impact on your experience.
+We now publish the chart's docker images to both [Quay.io] and [Docker Hub] and
+the chart is from now configured to use the images at Quay.io by default.
+Previous releases of images (excluding pre-releases) has been copied over to
+Quay.io as well.
 
-For the benefit of people running older versions of z2jh and are throttled by dockerhub,
-we have actually copied all our _released_ images from Docker Hub to Quay.io as well.
-So you can opt in to using the images from Quay.io with the following config:
```

Comment:
Users won't be throttled as it is now, so I figure its better to not mention this.